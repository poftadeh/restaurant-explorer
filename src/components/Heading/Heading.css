.main-heading {
  font-weight: 900;
  font-size: 5rem;
}