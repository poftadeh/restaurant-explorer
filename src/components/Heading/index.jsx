import React from 'react';

const Heading = () => {
  return <h1>Restaurant Finder</h1>;
};

export default Heading;
