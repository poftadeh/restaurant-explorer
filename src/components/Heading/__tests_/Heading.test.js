import React from 'react';
import { shallow } from 'enzyme';
import Heading from '../index';

it('renders <Heading /> properly', () => {
  const wrapper = shallow(<Heading />);
  expect(wrapper).toMatchSnapshot();
});
