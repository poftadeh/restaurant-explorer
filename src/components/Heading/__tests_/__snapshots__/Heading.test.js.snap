// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Heading /> properly 1`] = `ShallowWrapper {}`;
